distributed/client.py

```diff
@@ -4379,8 +4380,8 @@ def update(self, futures):
         The added futures will emit from the iterator once they finish"""
         with self.lock:
             for f in futures:
-                if not isinstance(f, Future):
-                    raise TypeError("Input must be a future, got %s" % f)
+                if not isinstance(f, Future) and not isinstance(f, ActorFuture):
```

Comment:
```suggestion
                if not isinstance(f, (Future, ActorFuture):
```

Reply:
Actually we should probably keep this as two if statements, and maybe include a second `_track_actorfuture` method

Reply:
I think this is now done (please see ff62b0b).